pkg/core/native/native_test/neo_test.go

```diff
@@ -771,6 +771,36 @@ func TestNEO_CalculateBonus(t *testing.T) {
 	})
 }
 
+func TestNEO_UnclaimedGas(t *testing.T) {
+	// Initialize the test invoker for NEO validators.
+	neoValidatorsInvoker := newNeoValidatorsClient(t)
+	e := neoValidatorsInvoker.Executor
+
+	// Create a new account with no NEO balance.
+	acc := neoValidatorsInvoker.WithSigners(e.NewAccount(t))
+	accH := acc.Signers[0].ScriptHash()
+
+	t.Run("non-existing account", func(t *testing.T) {
+		// non-existing account, should return zero unclaimed GAS.
+		acc.Invoke(t, 0, "unclaimedGas", util.Uint160{}, 1)
+	})
+
+	t.Run("zero balance", func(t *testing.T) {
+		acc.Invoke(t, 0, "unclaimedGas", accH, 1)
+	})
+
+	t.Run("unclaimed GAS", func(t *testing.T) {
+		amount := int64(100)
+		neoValidatorsInvoker.Invoke(t, true, "transfer", e.Validator.ScriptHash(), accH, amount, nil)
+
+		for range 10 {
+			e.AddNewBlock(t)
+		}
+		acc.Invoke(t, 550, "unclaimedGas", accH, e.Chain.BlockHeight()+1)
```

Comment:
@AnnaShaleva, can the amount of unclaimed gas after 10 blocks be calculated (is there any formula)?

Reply:
Sure, it's exactly https://github.com/nspcc-dev/neo-go/blob/9a383608244a822772b9e6cea25846ddd2102b92/pkg/core/native/native_neo.go#L768

A part of reward is distributed for NEO holding and a part of reward is distributed for voting, but for your test-case it's only NEO holding. And see also the tail of `TestNEO_CalculateBonus`.